src/odemis/gui/cont/tabs.py

```diff
@@ -4391,7 +4395,475 @@ def _on_align_move_done(self, f=None):
 
     @classmethod
     def get_display_priority(cls, main_data):
-        if main_data.role in ("secom", "enzel"):
+        if main_data.role in ("secom",):
+            return 1
+        else:
+            return None
+
+
+class EnzelAlignTab(Tab):
+    """
+    Tab to perform the 3 beam alignment of Enzel so that the FIB, SEM and FLM look at the same point.
+    """
+    def __init__(self, name, button, panel, main_frame, main_data):
+        tab_data = guimod.EnzelAlignGUIData(main_data)
+        super(EnzelAlignTab, self).__init__(name, button, panel, main_frame, tab_data)
+        self.set_label("ALIGNMENT")
+        self._stream_controllers = []
+        self._stage = main_data.stage
+        self._stage_global = main_data.stage_global
+        self._aligner = main_data.aligner
+
+        # Check stage FAV positions in its metadata, and store them in respect to their movement
```

Comment:
I don't think the second part of the comment is actually done, is it?

Reply:
No the storing part isn't done and actually should be removed from the comment.